### PR TITLE
fix(useposition): silences uselayouteffect ssr warning

### PR DESCRIPTION
**Pull request by dzucconi**
[Shout out](https://medium.com/@alexandereardon/uselayouteffect-and-ssr-192986cdcf7a)

### CODE REVIEW

File: packages/palette/src/utils/usePosition.ts [hunk 1]
```diff
@@ -44,7 +45,7 @@ export const usePosition = ({
   const tooltipRef = useRef<HTMLElement | null>(null)
   const anchorRef = useRef<HTMLElement | null>(null)
 
-  useLayoutEffect(() => {
```
Comment (damassi) [inline packages/palette/src/utils/usePosition.ts:47]: Or maybe just use `useEffect`? Is there a noticeable difference between the two UX-wise?

File: packages/palette/src/utils/usePosition.ts [hunk 2]
```diff
@@ -44,7 +45,7 @@ export const usePosition = ({
...
+  useIsomorphicLayoutEffect(() => {
```
Comment (damassi) [inline packages/palette/src/utils/usePosition.ts:48]: Or maybe just use `useEffect`? Is there a noticeable difference between the two UX-wise?

Comment (dzucconi) [reply packages/palette/src/utils/usePosition.ts:48]: Oh, yeah I'm intentionally using useLayoutEffect here. It's kind of wild actually; the return from `getBoundingClientRect` is incorrect for long enough for the position to be wrong. You see it show up as a very brief flash as the popover repositions itself.

Comment (damassi) [reply packages/palette/src/utils/usePosition.ts:48]: Cool 👍 Figured as much but wanted to be sure

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette@14.43.1`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4014.43.1) :rocket:

**Pull request merged**